doc/source/serve/production.md

```diff
@@ -583,3 +565,218 @@ This section summarizes the best practices when deploying to production:
+
+A [RayService] CR encapsulates a multi-node Ray Cluster and a Serve application that runs on top of it into a single Kubernetes manifest.
+Deploying, upgrading, and getting the status of the application can be done using standard `kubectl` commands.
+This section walks through how to deploy, monitor, and upgrade the [`FruitStand` example](serve-in-production-example) from above on Kubernetes.
+
+### Installing the KubeRay operator
+
+This guide assumes that you have a running Kubernetes cluster and have `kubectl` configured to run commands on it.
+See the [Kubernetes documentation](https://kubernetes.io/docs/setup/) if you need help getting started.
+
+The first step is to install the `KubeRay` operator into your Kubernetes cluster.
+This creates a pod that runs the `KubeRay` controller to manage resources based on the `RayService` CRs you create.
+
+Install the operator using `kubectl apply` and check that the controller pod is running:
+```console
+$ kubectl create -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources?ref=v0.3.0"
+$ kubectl apply -k "github.com/ray-project/kuberay/manifests/base?ref=v0.3.0"
+$ kubectl get deployments -n ray-system
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE
+ray-operator   1/1     1            1           40s
+
+$ kubectl get pods -n ray-system
+NAME                            READY   STATUS    RESTARTS   AGE
+ray-operator-75dbbf8587-5lrvn   1/1     Running   0          31s
+```
+
+For more details, see the [KubeRay installation guide](https://ray-project.github.io/kuberay/deploy/installation/).
+
+### Deploying a Serve application
+
+Once the KubeRay controller is running, you can manage your Ray Serve application by creating and updating a `RayService` CR.
+`RayService` custom resources consist of the following:
+- a `KubeRay` `RayCluster` config defining the parameters for the cluster that the Serve application runs on.
+- a Ray Serve [config](serve-in-production-config-file) defining the Serve application to run on the cluster.
+
+When the `RayService` is created, the `KubeRay` controller first creates a Ray cluster using the provided configuration.
+Then, once the cluster is running, it deploys the Serve application to the cluster using the [REST API](serve-in-production-deploying).
+The controller also creates a Kubernetes Service that can be used to route traffic to the Serve application.
+
+Let's see this in action by deploying the same [`FruitStand` example](serve-in-production-example) from above.
+The Serve config for the example is embedded into [this example `RayService` CR](https://github.com/ray-project/kuberay/blob/release-0.3/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml).
+To follow along, save this CR locally in a file named `ray_v1alpha1_rayservice.yaml`.
```

Comment:
Provide a bash command to download the file to the right name.

Reply:
good idea, added